coreTable/OwidTableConstants.ts

```diff
@@ -34,6 +38,7 @@ export interface OwidColumnDef extends CoreColumnDef {
     coverage?: string
     datasetId?: string
     datasetName?: string
+    display?: LegacyVariableDisplayConfigInterface
```

Comment:
Does `CoreColumnDef` already have this?

Reply:
Oh great! Thanks. So much more inheritance in our codebase now haha.